resources/authgear/templates/en/web/__header.html

```diff
@@ -1,11 +1,20 @@
 {{ define "__header.html" }}
+
+{{ $light := call $.StaticAssetURL "app-logo" }}
+{{ $dark := call $.StaticAssetURL "app-logo-dark" }}
+{{ if not $dark }}
```

Comment:
I think we may not need js for that this feature? And only assign ` $dark = $light` when `not $dark or not $.DarkThemeEnabled`?

Reply:
Updated the approach to not use JS at all.